### ISSUE TITLE
(Win) No Stack/Variable display bug

### ISSUE THREAD

**Issue by chrsm** [Kindari/SublimeXdebug#15]
Hello! Awesome plugin, this could be really useful if it showed me the current stack =)

Here's a trace:

```
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 356, in run_
  File ".\Xdebug.py", line 401, in run
  File ".\Xdebug.py", line 420, in callback
AttributeError: 'NoneType' object has no attribute 'current'
....later....
Traceback (most recent call last):
  File ".\Xdebug.py", line 420, in callback
AttributeError: 'NoneType' object has no attribute 'current'
```

Things still work, such as setting a breakpoint, 'running' after that breakpoint up until another breakpoint..just can't see the current stack =)

**Comment by jotson** [comment]
I think it's trying to open the appropriate file for debugging but can't find it on your file system. Is there anything _unusual_ about the file? Like being symlinked or is it on a network share or anything like that? Try adding this code to the plugin just before line 418 which reads <code>sublime.status_message('Xdebug: breakpoint')</code> to get some more information. Use spaces to make sure the new lines are aligned with the line below it or the code won't run properly:

```
            print '>>>break ' + child.getAttribute('filename') + ':' + child.getAttribute('lineno')
            print res.toprettyxml()
```

Quit Sublime. Then run the debug session again until you get the trace and send me all of the output in the console. The '>>>break' line you'll see in the console will show the path of the file that Sublime is trying to open. Let me know if that gives you any clues.

**Comment by chrsm** [comment]
> > > break file:///E:/src/**projectName**/index.php:9
> > >     <response command="run" reason="ok" status="break" transaction_id="2" xmlns="urn:debugger_protocol_v1"      xmlns:xdebug="http://xdebug.org/dbgp/xdebug">
> > >         <xdebug:message filename="file:///E:/src/__projectName__/index.php" lineno="9"/>
> > >     </response>

It's the file that is currently opened and the line in which the breakpoint exists.

*\* edit *\* - I forgot to submit this so I guess an edit notice isn't necessary. Anyways.. Let me be the first to say that I only dabble in python, so I could be 100% wrong here.

I started tracing down to show_file() around line 600. There are two vars created out of any given uri:

transport
file

It seems that when a uri like file:///c:/what/ever/index.php gets parsed in this way, it ends up splitting into:

file
/E:/what/ever/index.php

The first / on the actual file is probably the issue.

**Comment by chrsm** [comment]
Yup, this is the issue.

I went ahead and added a line:
filename = filename.replace('/', '', 1)

After the uri.split in show_file(). I do not know how to do much in python, so I'm not sure how to do it in any better of a way. Thanks for the awesome plugin! =)

**Comment by jotson** [comment]
Thanks. This should be fixed in the latest commit SHA 6cab2cba7d11c2c0a29bc4c83107a71ed7a01f75

**Comment by chrsm** [comment]
Awesome, thank you!

**Comment by joetech** [comment]
I'm getting this same issue, but the problem is more likely that it's trying to look locally for the file using it's remote path.  My development site is on a remote server, xdebug is triggered and listening, but generates the AttributeError when it hits the xdebug_break(); I added into the code (adding breakpoints in Sublime Text 2 does nothing).

Additionally (and potentially related), on starting the debugger, I'm notified that the url is not present in the project settings even though I've set it per the instructions.

I've loaded up an independent xdebug client and it steps through perfectly, so xdebug is working.  I just can't get this plugin to play nice with it.  Any help would be awesome.
